source/common/config/utility.h

```diff
@@ -219,6 +219,20 @@ class Utility {
     return config;
   }
 
+  template <class ProtoMessage, class Factory>
```

Comment:
I don't think you need the first template parameter, it is always Struct.

Reply:
was being consistent with the other translate function but will update!